pkg/kubelet/certificate/kubelet.go

```diff
@@ -278,18 +278,20 @@ type kubeletServerCertificateDynamicFileManager struct {
 	keyFile                   string
 	dynamicCertificateContent *dynamiccertificates.DynamicCertKeyPairContent
 	currentTLSCertificate     atomic.Pointer[tls.Certificate]
+	ctx                       context.Context
```

Comment:
Storing context in a structure instead of passing as a parameter to functions considered anti-pattern in go, please avioid.